openstack_controller/metadata.csv

```diff
@@ -1,5 +1,6 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 openstack.cinder.response_time,gauge,,millisecond,,Duration that an HTTP request takes to complete when making a request to cinder endpoint,0,openstack_controller,cinder response time,
+openstack.cinder.snapshot.count,gauge,,,,Number of cinder snapshots,0,openstack_controller,cinder snapshot ct,
```

Comment:
Could you review if `integer` and `boolean` metrics from [here](https://docs.openstack.org/api-ref/block-storage/v3/index.html#list-snapshots-and-details) would be interesting to report?

Reply:
I think that you are looking in the request parameters instead of the response parameters. The only int or bool response parameter that we aren't already tracking is `consumes_quota`, which based on its description seems to have a very limited use case. However, similar to what you asked with volumes, I have converted the size tag into a metric.